app/scripts/views/components/dataTable/tableView.js

```diff
@@ -78,7 +80,9 @@
 
       return result;
     }
-    
+
+    content.dataForDownload = {};
```

Comment:
I think dataForDownload is initialized in generalChart.js, then why to initialize in every chart implementation?